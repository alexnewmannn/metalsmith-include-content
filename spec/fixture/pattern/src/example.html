<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Example Test</title>
</head>
<body>
<p>This example show that content can be included using include path.html</p>
    includeContentFrom nav.html

    includeContentFrom missing.html
</body>
</html>
