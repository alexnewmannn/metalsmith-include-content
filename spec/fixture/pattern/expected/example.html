<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Example Test</title>
</head>
<body>
<p>This example show that content can be included using include path.html</p>
    <nav>
    <ul>
        <li>Link</li>

        <li>Link</li>

        <li>Link</li>

    </ul>
</nav>


    "missing.html" file not found.
</body>
</html>
